controllers/grid/catalogEntry/form/PublicationFormatForm.php

```diff
@@ -48,7 +48,8 @@ class PublicationFormatForm extends Form
      */
     public function __construct($monograph, $publicationFormat, $publication)
     {
-        parent::__construct('controllers/grid/catalogEntry/form/formatForm.tpl');
+        $localeNames = Application::get()->getRequest()->getContext()->getSupportedSubmissionMetadataLocaleNames() + $publication->getLanguageNames();
```

Comment:
Shouldn't this be similar to OJS i.e. using `submission->getPublicationLanguageNames()` ?